cves/CVE-2006-5752.yml

```diff
@@ -48,14 +53,19 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: a44b079370ea0d4fb5748dc510d9b14698a196cc
-     note:
+     note: specified a charset when setting content type 
    - commit: 6f0c8008746b58eb0a59f59501163eea2921d03d
-     note:
+     note: specified a charset when setting content type
 vccs:
   - commit: dba4852ebca38b6c58f7600e4ef0d57bd9c81a8f
-    note:
-  - commit:
-    note:
+    note:  
```

Comment:
I'm assuming that this vcc was already provided for you, but you should still leave a note about why it was listed.

Reply:
removed vcc as it did not have any relation to the vulnerability introduced.